### PR TITLE
Fix/10498 Show New Functions Screen footernote only with delta onboarding

### PR DESCRIPTION
**Pull request by puneetmahali1**
## Description
The footernote cell on New Functions Screen was displaying at the time of delta onboarding and after that navigate from App Information. Now, it should display only with the delta onboarding.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10498

## Screenshots

https://user-images.githubusercontent.com/72390277/196912150-426f0d06-b5ba-4f9b-825d-55d51b1cbc12.mov

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift [hunk 1]
```diff
@@ -11,15 +11,17 @@ class AppInformationViewController: DynamicTableViewController, NavigationBarOpa
 	
 	init(
 		elsService: ErrorLogSubmissionProviding,
+		store: Store,
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift:14]: Do we have a better option rather an injecting store here and passing everywhere?

Comment (puneetmahali1) [reply src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift:14]: Unfortunately, I didn't find the better option!

**Review by naveeddotio — CHANGES_REQUESTED** [review]
Please fix this as discussed

**Review by 30mar — APPROVED** [review]
No comments except what naveed said

**Comment by puneetmahali1** [comment]
> 

Thanks, Done!

**Comment by puneetmahali1** [comment]
> Please fix this as discussed

Thanks, Done!

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**